Packs/Slack/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Slack",
     "description": "Interact with Slack API - collect logs, send messages and notifications to your Slack team.",
     "support": "xsoar",
-    "currentVersion": "3.1.46",
+    "currentVersion": "3.1.47",
```

Comment:
@amshamah419 are 4 new commands worth bumping to 3.2?

Reply:
3

Reply:
still, new feature (rather than bug fixes)

Reply:
We can bump to 3.2.0. There really isn't anything against doing so and having a minor bump would help us differentiate the versions quickly if there's an issue.